content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,112 @@
+# HOW-TO: Build your own Collector instance
+
+If you are planning to build and debug your components, you are going to need your own Collector instance. That will allow you to launch and debug your Collector components directly within your favorite Golang IDE.
+
+The other really cool aspect of approaching the component development this way is that you can use all the cool debugging features from your IDE (stack traces are great teachers!) to understand how the Collector itself interacts with your component code.
```

Comment:
Remove the second "cool"

Reply:
Hi @svrnm, fixed on [7d3d01](https://github.com/open-telemetry/opentelemetry.io/pull/1277/commits/7d3d01bd356ff8c6ae72ffd52cc476f69108f7ca)